web/AngularSPA/PetSupport/tailwind.config.js

```diff
@@ -8,19 +8,188 @@ module.exports = {
   },
   darkMode: 'class', // or 'media' or 'class'
   theme: {
+    screens: {
+      'sm': {'min': '640px', 'max': '767px'},
+      'md': {'min': '768px', 'max': '1023px'},
+      'lg': {'min': '1024px', 'max': '1279px'},
+      'xl': {'min': '1280px', 'max': '1535px'},
+      '2xl': {'min': '1536px'},
+    },
     colors: {
       transparent: 'transparent',
       current: 'currentColor',
       black: colors.black,
       white: colors.white,
-      gray: colors.trueGray,
       indigo: colors.indigo,
-      green: colors.green,
       red: colors.rose,
-      yellow: colors.amber,
-      "primary": "#219653"
-
+      yellow: { 
+        DEFAULT: "#F2C94C",
+        dark: "#F8C21D",
+        light: "#F3D473",
+        lightest: "#F3DD98",
+        normal: colors.amber,
+      },
+      green: {
+        DEFAULT: "#219653",
+        dark: "#046A2F",
+        light: "#49B678",
+        lightest: "#8ACBA6",
+        normal: colors.green,
+      },
+      gray: {
+        DEFAULT: "#848484",
+        darkest: "#4A4A4A",
+        light: "#B9B9B9",
+        lightest: "#D4D4D4",
+        normal: colors.trueGray,
+      }
+    },
+    fontFamily: {
```

Comment:
I've checked that most of this properties under colors are default tailwind settings. Remove it please ;)